Sources/OpenTracing/Format.swift

```diff
@@ -3,24 +3,47 @@ import Foundation
 /// Format and carrier for extract(). Marker protocol.
 public protocol FormatReader {}
 
-// Format and carrier for inject(). Marker protocol.
+/// Format and carrier for inject(). Marker protocol.
 public protocol FormatWriter {}
 
+/// Read interface for a textmap
 public protocol TextMapReader: FormatReader {
+
+    /// Return the text map as a [String: String]
+    ///
+    /// - returns: textmap as a [String: String] representation
     func getAll() -> [String: String]
 }
 
+/// Write interface for a textmap
 public protocol TextMapWriter: FormatWriter {
+
+    /// Set all values from a [String: String]
+    ///
+    /// - parameter textMap: dictionary containing key-value pairs to set into the text map
     func setAll(textMap: [String: String])
 }
 
+/// Read interface for HTTP headers
```

Comment:
Maybe elaborate on what these are used for? Same with all other protocols (won't comment individually)